add json schema to core

MarkSackerberg opened the pull request.

L0STE commented on src/pages/core/json-schema.md:
```diff
@@ -0,0 +1,227 @@
+description: Understand the structure and requirements of the off-chain JSON schema for Metaplex Core assets.
+---
+
+The off-chain JSON metadata for Metaplex Core assets is similar to the Metaplex Token Metadata standard. However, since more data can be stored on-chain in the asset itself using plugins, some of the data, like creators, are no longer part of the JSON standard. Although they are not required, they can still be used, and some wallets and tools might render them.
```
```suggestion
The off-chain JSON metadata for Metaplex Core assets resembles the Metaplex Token Metadata one. However, with plugins allowing more data to be stored on-chain, fields like creators are now optional but may still be included, as some wallets and tools may render them.
```

L0STE commented on src/pages/core/json-schema.md:
```diff
@@ -0,0 +1,227 @@
+
+If you miss some fields that you knew from Metaplex Token Metadata those are probably deprecated. The `creators` for example are part of the [Royalties Plugin](core/plugins/royalties) now.
+
+### Required Fields
```
Do you think wrapping this section up in a table like in [this section](https://developer-hub-git-core-schma-metaplex-foundation.vercel.app/core/guides/javascript/how-to-create-a-core-nft-asset-with-javascript#uploading-the-metadata) might make it a little more clean? (For both Required and Optional field)

L0STE commented on src/pages/core/json-schema.md:
```diff
@@ -0,0 +1,227 @@
+
+## JSON Schema Fields
+
+Now, we will explore the various fields in the Schema. Examples for different types of Assets such as Videos, Images, 3D Files, and HTML Assets can be found at the bottom of the page. It's important to note that all of these different types can also be part of a single Asset using the `image`, `animation_url`, and `properties` fields.
```
I think we can put this text where the example are!

tonyboylehub reviewed (commented):
Can we possibly add a copyable blank template towards the start maybe? I feel like that might be good to have here.

MarkSackerberg commented:
> Can we possibly add a copyable blank template towards the start maybe? I feel like that might be good to have here.

I've changed the order to have the examples at the top.

Also addressed the other nitpicks. Feel free to have a last look @tonyboylehub @L0STE
